mix.exs

```diff
@@ -0,0 +1,29 @@
+defmodule HelloElixir.MixProject do
+  use Mix.Project
+
+  def project do
+    [
+      app: :hello_elixir,
+      version: "0.1.0",
+      elixir: "~> 1.12",
+      start_permanent: Mix.env() == :prod,
+      deps: deps()
+    ]
+  end
+
+  # Run "mix help compile.app" to learn about applications.
+  def application do
+    [
+      extra_applications: [:logger]
+    ]
+  end
+
+  # Run "mix help deps" to learn about dependencies.
+  defp deps do
+    [
+      {:credo, "~> 1.6", only: [:dev, :test], runtime: false}
```

Comment:
I am not sure if `credo` makes any  sense in `:test` environment. Also, this is a premature addition, let’s add code when we understand whys and whats.